nodejs/private/nodejs_repo_host_os_alias.bzl

```diff
@@ -18,20 +18,13 @@ alias(name = "node_files",              actual = "@{node_repository}_{os_name}//
 alias(name = "npm_files",               actual = "@{node_repository}_{os_name}//:npm_files")
 exports_files([
     "index.bzl",
-    "bin/node{ext}",
 ])
 """.format(
         node_repository = repository_ctx.attr.user_node_repository_name,
         os_name = os_name(repository_ctx),
         ext = ext,
     ))
 
-    repository_ctx.symlink("../{node_repository}_{os_name}/bin/node{ext}".format(
```

Comment:
Is anything testing this? No tests require changing or update? :/

Reply:
Nothing needs updating. It was legacy and unused.